packages/fast-jss-manager-react/src/jss-manager.spec.tsx

```diff
@@ -60,201 +72,127 @@ const staticAndDynamicStyles: ComponentStyles<any, any> = {
 };
 
 describe("The JSSManager", (): void => {
-    interface TestDesignSystem {
-        color: string;
+    class NoPropsManager extends JSSManager<any, any, any> {
+        protected styles: void = undefined;
+        protected managedComponent: React.ComponentType<any> = SimpleComponent;
     }
 
-    function renderChild(): string {
-        return "children";
+    class StyledManager extends JSSManager<any, any, any> {
+        protected styles: any = staticAndDynamicStyles;
+        protected managedComponent: React.ComponentType<any> = StyledComponent;
     }
 
-    // JSS doesn't export their StyleSheet class, so we can compile a stylesheet and
-    // access it's constructor to get a reference to the StyleSheet class.
-    const StyleSheet: any = jss.createStyleSheet({}).constructor;
-
-    const stylesheet: any = {
-        class: {
-            color: (config: TestDesignSystem): string => {
-                return config.color;
-            },
-        },
-    };
-
-    const testDesignSystem: TestDesignSystem = {
-        color: "red",
-    };
+    test("should render the managedComponent", () => {
+        const rendered: any = mount(<NoPropsManager />);
 
-    function functionStyleSheet(config: TestDesignSystem): any {
-        return {
-            class: {
-                color: config.color,
-            },
-        };
-    }
-
-    test("should not throw when no stylesheet is provided", (): void => {
-        expect(
-            (): void => {
-                mount(<JSSManager render={renderChild} />);
-            }
-        ).not.toThrow();
+        expect(rendered.find("div")).toHaveLength(1);
+        expect(rendered.find("SimpleComponent")).toHaveLength(1);
     });
 
-    test("should not throw when no stylesheet is provided and design system is changed", (): void => {
-        const rendered: ReactWrapper = mount(
-            <JSSManager designSystem={testDesignSystem} render={renderChild} />
-        );
+    test("should pass props to managed component", (): void => {
+        const rendered: any = mount(<NoPropsManager id="id" />);
 
-        expect(
-            (): void => {
-                rendered.setProps({ designSystem: { color: "blue" } });
-            }
-        ).not.toThrow();
+        expect(rendered.find("SimpleComponent").prop("id")).toBe("id");
     });
 
-    test("should compile a stylesheet when mounting", (): void => {
-        const objectStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                render={renderChild}
-                designSystem={testDesignSystem}
-                styles={stylesheet}
-            />
-        );
-
-        const functionStylesheetComponent: ReactWrapper = mount(
-            <JSSManager
-                render={renderChild}
-                styles={functionStyleSheet}
-                designSystem={testDesignSystem}
-            />
+    test("should not pass the jssStyleSheet prop to managed component", (): void => {
```

Comment:
should this be "should not pass the jssStyleSheet prop to a managed component without styles"?